components/Input/input.css

```diff
@@ -0,0 +1,39 @@
+:root {
+    --error-color: rgb(255, 0, 0);
+    --success-color: rgb(0, 128, 0);
+}
+
+.inp-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin-top: 1rem;
+}
+
+label {
+    font-size: 1.2em;
+}
+
+input[type="text"] {
+    padding: 1rem;
+    width: 20rem;
+    border-radius: 4px;
+}
+
+.error {
+    color: var(--error-color);
+}
+
+ .inp-container > .error-inp {
+    border: 2px solid var(--error-color);
+    outline: none;
+}
+
```

Comment:
Just if you are not aware, you can make whole border itself a variable, so if you want to change thickness of border that also can be done. E.g. `border: var(--error-border); ` define error border like `--error-border: 2px solid #000`